.github/workflows/deploy-preview-cleanup.yml

```
@@ -55,17 +55,36 @@ jobs:
         with:
           pr_number: ${{ github.event.inputs.pr_number || github.event.number }}
 
+      - name: Install Flyctl
+        uses: superfly/flyctl-actions/setup-flyctl@master
+
       - name: 🗑️ Destroy PR app in Fly.io
         id: destroy_app
-        continue-on-error: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.force_cleanup == 'true' }}
-        uses: superfly/fly-pr-review-apps@1.5.0
+        continue-on-error: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.force_cleanup ==
+        'true' }}
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 64: could not find expected ':'

(syntax-check)